JS Fundamentals Assignment(optional)/Q2_Optional.txt

```diff
@@ -0,0 +1,50 @@
+
+let objA = {
+    name:"Harry Potter",
+  age: 12,
+  address: {
+  details: ["4", "Privet Drive"],
+  area:"Little Whinging",
+  city: "Surrey",
+  state: "England"
+  } 
+}
+
+
+function deepCopy(objA) {
```

Comment:
One issue in this function. Even for arrays it is creating objects. For above example for details it would create {0: "4", 1: "Privet Drive"}